src/VSPackage.cs

```diff
@@ -189,8 +203,22 @@ IEnumerable<ProjectItem> GetProjectFiles(Project proj)
                 ResourceAdornmentManager.TextForegroundColor = _textColor;
 
                 var plural = ResourceAdornmentManager.ResourceFiles.Count > 1 ? "s" : string.Empty;
-                (await this.GetServiceAsync(typeof(DTE)) as DTE).StatusBar.Text = $"String Resource Visualizer initialized with {ResourceAdornmentManager.ResourceFiles.Count} resource file{plural}.";
+                dte.StatusBar.Text = $"String Resource Visualizer initialized with {ResourceAdornmentManager.ResourceFiles.Count} resource file{plural}.";
             }
         }
     }
+
+    static class TaskExtensions
+    {
+        internal static void LogAndForget(this Task task, Type type)
+        {
+            task.ContinueWith(t =>
+            {
+                if (t.Exception != null)
+                {
+                    VsShellUtilities.LogError(type.FullName, t.Exception.ToString());
+                }
+            });
```

Comment:
Don't call `ContinueWith` without specifying a `TaskScheduler`. In this case, you should be running on the UI thread since you're filing to the activity log and I'm not sure that's a free-thread operation.

Also, you can specify `TaskContinuationOptions.OnlyOnFaulted` and then you can drop your `if` check, and your continuation won't even execute if no error occurs, making it more efficient.

If you also pass `type` in as the `state` parameter to `ContinueWith` and use that state arg in your delegate, you can altogether avoid a closure and delegate allocation with each run of this method, making it *almost* free of runtime cost.

Consider installing the [VSSDK Analyzers](https://aka.ms/vssdkanalyzers) which will catch issues like this.